azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/java/com/microsoft/azure/toolkit/lib/mysql/model/MySqlServerConfig.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.lib.mysql.model;
+
+import com.microsoft.azure.toolkit.lib.common.model.Region;
+import com.microsoft.azure.toolkit.lib.common.model.ResourceGroup;
+import com.microsoft.azure.toolkit.lib.common.model.Subscription;
+import com.microsoft.azure.toolkit.lib.database.entity.IDatabaseServerEntity;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.Objects;
+
+@Getter
+@Builder
+public class MySqlServerConfig implements IDatabaseServerEntity {
+
```

Comment:
if it is the same with SqlServerConfig, can we use the same class?

Reply:
yes. as files of different resources domain, I prefer to keep them as different classes